hasura/metadata/databases/default/tables/public_feedbacks.yaml

```diff
@@ -0,0 +1,3 @@
+table:
```

Comment:
we can remove this too, right?

Reply:
No, it was being used by the Prisma code in the scripts, so we need to track it in Hasura so we can interact with it via graphql instead